docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -0,0 +1,34 @@
+= Getting Started with APPUiO Cloud
+
+This tutorial describes your first steps with APPUiO Cloud, from signup to your first deployment.
+
+== Step 1: Create an Account
+
+To create your APPUiO Cloud account, fill the form https://www.appuio.ch/en/offering/cloud/[in this page]. You will receive an e-mail inviting you to create a password for your newly created account.
+
+== Step 2: Login
+
+Login to the https://control.vshn.net/[VSHN Control Panel] following the instructions in your welcome e-mail. You will be asked to
+
+== Step 3: Choose a Zone
+
+In the https://control.vshn.net/appuio-cloud[APPUiO Cloud] page, select the link that corresponds to your preferred zone.
```

Comment:
Better point to https://docs.appuio.cloud/user/references/zones.html for the time being. Once the APPUiO Cloud Portal is ready, we should update this and point to the new Portal.

Reply:
The thing is, when I signed up for APPUiO Cloud, I got an account to the Control Panel, where this information is available. In terms of "getting started" user experience, the official list of zones should be the one in the Control Panel.

Reply:
The official list will be in the APPUiO Cloud Portal. But as we don't have it yet, let's leave it like this.